Web/ForgeTrust.AppSurface.Docs/wwwroot/css/app.css

```
@@ -1833,9 +1833,13 @@ body {
     border: 1px solid var(--docs-color-border-default-heavier);
     border-radius: 0.35rem;
     background: var(--docs-color-surface-panel);
+    -webkit-box-decoration-break: clone;
+    box-decoration-break: clone;
     padding: 0.08rem 0.32rem;
     color: var(--docs-color-text-info);
     font-size: 0.92em;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)